InnoClinic.OfficesAPI/DAL/Entities/Office.cs

```diff
@@ -0,0 +1,27 @@
+﻿using MongoDB.Bson;
+
+namespace DAL.Entities;
+
+public sealed class Office
+{
+    public ObjectId Id { get; set; }    
+    public byte[]? Photo { get; set; }  
+    public string City { get; set; }
+    public string Street { get; set; }
+    public string HouseNumber { get; set; }
+    public string OfficeNumber { get; set; }
+    public string RegistryPhoneNumber { get; set; } 
+    public bool Status { get; set; }
```

Comment:
`Status=true` What does it mean? Better to rename it or use another type

Reply:
According to doc, this `Status` thould be either `Active` or `Inactive`. So may be better to name it like `IsActive` or something like that